sdk/iot/hub/samples/src/generate_certificate.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
can we rename this file to test_generate_certificate.sh
So that we make sure that it is for testing ?

Reply:
for naming parity how about sample_generate_certificate.sh

Reply:
If we decided to change the name, please don't forget to mirror the change in Provisioning and docs.